### ISSUE TITLE
Encourage usage of CSL-JSON

### ISSUE THREAD

**Issue by mariusbarth** [crsh/papaja#387]
As can be seen in [this pull request](https://github.com/rstudio/rmarkdown/pull/1819), it is best to use CSL-JSON compared to BibTeX. Does this also apply to `papaja`, and should we also encourage usage of CSL-JSON in the documentation and the template?

**Comment by crsh** [comment]
Yes, I fully agree. This is also an issue I have considered for [`citr`](https://github.com/crsh/citr/issues/59). But due to limited time I haven't made any headway there because this will largely promoted by a major [change to the internals of `citr`](https://github.com/crsh/citr/issues/55). Once that change is implemented, `citr` will default to JSON files anyway. In short, sure.

How do you want to encourage the use of JSON? It seems this would be an issue for the manual, no?

**Comment by danieltomasz** [comment]
For example by  including as example "references.json" "instead references.bib"

**Comment by mariusbarth** [comment]
That's what I also had in mind. Unfortunately, it seems like the `.bib` file with references to R packages is created in a way that is not easily moved to `.json`, right?

**Comment by crsh** [comment]
Thanks for chiming in, Daniel. A quick thought: While CSL-JSON is preferred over BibTeX because it's not lossy, BibTeX is far more widely used and available. I fear that if we use CSL-JSON in the example, which is one of the first things a new user will look at, it may be off-putting. I know Zotero allows CSL-JSON export, but I think other commonly used reference managers (e.g., Citavi, Papers, Endnote) do/may not. But I would be happy to add a comment to the YAML that encourages use of JSON.

We should definitely mention this in the manual, though.

Marius, I think you are right. The references that we generate for the used R packages are only available in Bib(La)TeX-format, so we would need to convert them somehow.
